week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,7 +7,28 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1-  select * from rooms where rate > 100;
 
+2-  select * from reservations where checkin_date BETWEEN '2020-08-01' and '2020-09-01' and checkin_date + 3 < checkout_date;
+
+3-  select * from customers where city like 'M%';
+
+4- insert into room_types (room_type , def_rate) values ('PENTHOUSE', '185.00');
+
+5- insert into rooms  (room_no, rate, room_type) values ('501' ,'185.00', 'PENTHOUSE'), ('502' ,'185.00', 'PENTHOUSE');
+
+6- insert into rooms  (room_no, rate, room_type) values ('503' ,'143.00', 'PREMIER PLUS');
+
+7- SELECT count(distinct room_no) FROM reservations  WHERE checkin_date >= date_trunc('month', current_date - interval '1' month)   and checkout_date < date_trunc('month', current_date);  ===>  or SELECT count(room_no) FROM reservations  WHERE checkin_date >= date_trunc('month', current_date - interval '1' month);
```

Comment:
Good use of `count(distinct ...)` - excellent work, that's hard to find in the reading.